tech/languages/haskell/leksah.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Leksah
+subsection: haskell
+order: 4
+---
+
+# Leksah
+
+[Leksah](http://leksah.org/) is a Haskell IDE written in Haskell using Gtk. It offers Workspaces for complex project with multiple 
+packages with automatic build of dependencies. It enables standard IDE features (debugging, auto completition etc.).
+
+To install Leksah you need to install `ghc` and `cabal` first (or install the Haskell Platform).
```

Comment:
What about to write a sentence like:

To install Leksah run command:

```
sudo dnf install ghc cabal
```
 or install Haskell Platform by command:

```
sudo dnf install ....
```